### PR TITLE
Updated admin_peers to add protocol section 

### PR DESCRIPTION
**Pull request by MadelineMurray**
Updated admin_peers to add protocol section

### CODE REVIEW

File: docs/Reference/API-Methods.md [hunk 1]
```diff
@@ -360,28 +362,34 @@ Properties of the remote node object are:
...
-                 "pip/1"
-              ],
+                 "eth/64",
+                 "eth/65",
+                 "IBF/1"
+               ],
                "network": {
-                  "localAddress": "192.168.1.229:50115",
-                  "remoteAddress": "168.61.153.255:40303"
+                 "localAddress": "127.0.0.1:63139",
```
Comment (NicolasMassart) [inline docs/Reference/API-Methods.md:379]: not sure we should change the addresses to localhost. The previous example was better to help understand what you can get.

File: docs/HowTo/Find-and-Connect/Managing-Peers.md [hunk 2]
```diff
@@ -30,6 +30,16 @@ JSON-RPC API methods to monitor peer connections include:
...
+Each peer entry returned by [`admin_peers`](../../Reference/API-Methods.md#admin_peers) includes a
+`protocols` section. Use the information in the `protocols` section to:
+
+* Determine health of peers.  For example, an external process could use [`admin_peers`](../../Reference/API-Methods.md#admin_peers)
+and [`admin_removePeer`](../../Reference/API-Methods.md#admin_removepeer) to disconnect from peers that
+are stalled at a single difficulty for an extended period of time. 
+
+* Monitor node health. For example, if peers are reporting increasing difficulties but node
+is stuck at the same block number, the node may be on a different fork to most peers. 
+
```
Comment (shemnon) [inline docs/HowTo/Find-and-Connect/Managing-Peers.md:42]: maybe add
```

* Determine which protocol level peers are communicating with. For example, to see if `"version": 65` is being used to reduce transaction sharing traffic.
```

**Review by shemnon — APPROVED** [review]

File: docs/Reference/API-Methods.md (hunk 1 above)
Comment (MadelineMurray) [reply docs/Reference/API-Methods.md:379]: Updated

File: docs/HowTo/Find-and-Connect/Managing-Peers.md (hunk 2 above)
Comment (MadelineMurray) [reply docs/HowTo/Find-and-Connect/Managing-Peers.md:42]: Done

**Pull request merged**